### PR TITLE
add jest and cypress in Checkbox

### PR DESCRIPTION
**Pull request by slee1130**
### Purpose 🚀
_clear, one sentence description of primary purpose of this PR_

### Notes ✏️
_details of code change / secondary purposes of this PR_
- add jest and cypress testing in checkbox component
- feedback needed for cypress 
@oscarkwan 

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [x] MINOR (backward compatible) change to _these packages_
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/Checkbox/tests/Checkbox.spec.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    expect(getByRole("checkbox")).toBeInTheDocument();
+    expect(getByRole("checkbox")).toHaveAttribute("checked");
+  });
+
+  it("Renders checkedState is unchecked", () => {
+    const { getByRole } = renderComponent({ checkedState: checkboxStates.UNCHECKED });
+    expect(getByRole("checkbox")).not.toHaveAttribute("checked");
+  });
+
+  it("Renders checkedState is INDETERMINATE", () => {
```
Comment (oscarkwan) [inline packages/Checkbox/tests/Checkbox.spec.js]: no need to all caps the word

File: packages/Checkbox/tests/cypress/Checkbox.cypress.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+describe("<CheckBox />", () => {
+  before(() => {});
+
+  it("should be checked when you click the checkbox", () => {
+    cy.visitStorybook("checkbox-automation-tests--cypress");
+    cy.get("label").click();
+    cy.get("label").click();
```
Comment (oscarkwan) [inline packages/Checkbox/tests/cypress/Checkbox.cypress.js]: This text is actually not expecting any results. If you think we don't need a cypress test, and jest is sufficient then we don't need to add it here.  It seems like this is testing the same thing as jest

File: packages/Checkbox/tests/Checkbox.spec.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    a11yText: null,
+    checkedState: checkboxStates.CHECKED,
+    children: null,
+    isDisabled: false,
+    onChange: () => {},
+  };
+  return render(<Checkbox {...defaultProps} {...props} />);
+}
+
+describe("Checkbox", () => {
```
Comment (oscarkwan) [inline packages/Checkbox/tests/Checkbox.spec.js]: Can we add a couple more tests?

One for checking a box, and other checkboxes are also selected.
Another for disabled

File: packages/Checkbox/tests/cypress/Checkbox.cypress.js (hunk 2 above)
Comment (slee1130) [reply packages/Checkbox/tests/cypress/Checkbox.cypress.js]: ok

File: packages/Checkbox/tests/Checkbox.spec.js (hunk 1 above)
Comment (slee1130) [reply packages/Checkbox/tests/Checkbox.spec.js]: done

File: packages/Checkbox/stories/Checkbox.stories.js [hunk 4]
```diff
@@ -20,3 +20,5 @@ storiesOf("Checkbox", module)
       <CheckboxAlign />
     </Story>
   ));
+
+storiesOf("Checkbox/Automation Tests", module).add("Cypress", () => <ShowcaseStory />);
```
Comment (oscarkwan) [inline packages/Checkbox/stories/Checkbox.stories.js]: You also don't need this anymore

**Pull request merged**